src/gateways.ts

```diff
@@ -80,4 +80,5 @@ export default [
   'https://ipfs.tayfundogdas.me/ipfs/:hash',
   'https://ipfs.jpu.jp/ipfs/:hash',
   'https://ipfs.soul-network.com/ipfs/:hash',
+  'https://ipfs.twdragon.net/ipfs/:hash'
```

Comment:
```suggestion
  'https://ipfs.twdragon.net/ipfs/:hash',
```
@twdragon we need trailing commas now. I should probably add a pre-commit hook to this repo.